lib/node_modules/@stdlib/stats/base/dists/hypergeometric/README.md

```diff
@@ -108,7 +108,81 @@ var y = dist.cdf( 0.5 );
 var objectKeys = require( '@stdlib/utils/keys' );
 var hypergeometric = require( '@stdlib/stats/base/dists/hypergeometric' );
 
+// Print namespace contents
+console.log( 'Namespace contents:' );
 console.log( objectKeys( hypergeometric ) );
```

Comment:
Let's remove logging the namespace contents in case where we have a custom example.